pkgs/applications/virtualization/podman-desktop/default.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenv
+, yarn
+, nodejs-16_x
+, makeWrapper
+, fetchYarnDeps
+, desktopToDarwinBundle
+, fetchFromGitHub
+, fixup_yarn_lock
+, electron_23
+, makeDesktopItem
+,
+}:
+
+stdenv.mkDerivation (self: {
```

Comment:
Doesn't mkYarnPackage work?

Reply:
The code is generally migrated from the nix expression for `element-desktop`. I tried `mkYarnPackage` at first, but it seems to be triggering a bug in the dependency resolution of yarn. To be concise, the package `ws` resolved to a version that was not cached by `fetchYarnDeps`. Hence, I have to fix the lock file manually by applying a patch.